### PR TITLE
Azure Load Testing stable 1.0.0 changes

### PR DESCRIPTION
**Pull request by Harshan01**
# Description

* Re-add 2 client builders - `Administration` and `TestRun`
* Rename non-paged `List` functions to `Get`
* Rename `metricname` parameter to `metricName` in `ListMetrics` by removing `@Generated`

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by Harshan01** [comment]
@srnagar @weidongxu-microsoft Requesting review of the final changes based on APIView review

APIView review: https://apiview.dev/Assemblies/Review/cd72e6a5038d48558489e0380645fecb

File: sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/ReadmeSamples.java
Comment (srnagar) [inline sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/ReadmeSamples.java:39]: Same as above - don't need to have a builder variable and just call `buildClient()` after `endpoint()`

File: sdk/loadtesting/azure-developer-loadtesting/swagger/README.md
Comment (srnagar) [inline sdk/loadtesting/azure-developer-loadtesting/swagger/README.md:24]: Can this change me made directly in the swagger?

**Review by srnagar — APPROVED** [review]

File: sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunClient.java
Comment (weidongxu-microsoft) [inline sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunClient.java:264]: nit, this might actually benefit from a [x-ms-client-name](https://azure.github.io/autorest/extensions/#x-ms-client-name), either in readme.md if only for Java, or in swagger if for all SDK (instead of customization here).

Comment (Harshan01) [reply sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunClient.java]: Updated

File: sdk/loadtesting/azure-developer-loadtesting/swagger/README.md
Comment (Harshan01) [reply sdk/loadtesting/azure-developer-loadtesting/swagger/README.md:24]: Yes, we are currently discussing changing in swagger. However, it might take sometime as it is a breaking now since the docs have been published. For the time being, can this be handled using directives?

File: sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/ReadmeSamples.java
Comment (Harshan01) [reply sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/ReadmeSamples.java]: Done

File: sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/HelloWorld.java
Comment (Harshan01) [reply sdk/loadtesting/azure-developer-loadtesting/src/samples/java/com/azure/developer/loadtesting/HelloWorld.java]: Done

File: sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunClient.java
Comment (Harshan01) [reply sdk/loadtesting/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestRunClient.java]: Done

**Comment by weidongxu-microsoft** [comment]
I think all approved, let me know when you need it merged (resolve conflict first).

**Comment by Harshan01** [comment]
@weidongxu-microsoft failures seem to be related to management plane test case failure. Should that be resolved before we are allowed to merge data-plane?

**Comment by weidongxu-microsoft** [comment]
> @weidongxu-microsoft failures seem to be related to management plane test case failure. Should that be resolved before we are allowed to merge data-plane?

Just intermittent CI problem.

**Pull request merged**
